stories/navigation/UnstyledLink.stories.js

```diff
@@ -1,34 +1,40 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { action } from '@storybook/addon-actions';
 import StoryContainer from '../storyHelpers/StoryContainer';
+import { ThemeProvider } from '@sparkpost/matchbox/components/ThemeProvider';
+import { action } from '@storybook/addon-actions';
 
 import { UnstyledLink } from '@sparkpost/matchbox';
 
-class DemoWrapper extends React.Component {
-  render() {
-    return <a>{this.props.children}</a>;
-  }
+addDecorator(storyFn => (
+  <ThemeProvider>
+    <StoryContainer bg="white">{storyFn()}</StoryContainer>
+  </ThemeProvider>
+));
+
+export default {
+  title: 'Navigation|UnstyledLink',
+};
+
+function DemoWrapper(props) {
+  return <a>{props.children}</a>;
 }
 
-storiesOf('Navigation|UnstyledLink', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{ getStory() }</StoryContainer>
-  ))
+export const WithAnOnClickAction = withInfo()(() => (
+  <UnstyledLink onClick={action('Handle Click')}>A link</UnstyledLink>
+));
 
-  .add('with an on click action', withInfo()(() => (
-    <UnstyledLink onClick={action('Handle Click')}>A link</UnstyledLink>
-  )))
+export const WithAnExternalLink = withInfo()(() => (
+  <UnstyledLink to="https://google.com" external>
+    Google
+  </UnstyledLink>
+));
 
-  .add('with an external link', withInfo()(() => (
-    <UnstyledLink to='https://google.com' external>Google</UnstyledLink>
-  )))
-  
-  .add('with wrapper components', withInfo()(() => (
-    <>
-      <UnstyledLink component='button'>Any valid HTML tag</UnstyledLink>
-      <UnstyledLink component={({ children }) => <a>{children}</a>}>A Function</UnstyledLink>
-      <UnstyledLink component={DemoWrapper}>A Component</UnstyledLink>
-    </>
-  )));
+export const WithWrapperComponents = withInfo()(() => (
+  <>
+    <UnstyledLink component="button">Any valid HTML tag</UnstyledLink>
```

Comment:
I think I added this in at some point, but its a bad example for a link, since button restyles it